snapcraft/cli/env.py

```diff
@@ -50,6 +50,11 @@ def use_container(self):
     def remote(self):
         return self._remote
 
+    @property
+    def image(self):
+        # TODO: allow this to be overridden
```

Comment:
It slightly bugs me that now there's two places where the default image is defined... how about putting the default image in snapcraft/internal/common.py?

Reply:
Done